monai/networks/utils.py

```diff
@@ -52,6 +52,6 @@ def predict_segmentation(logits):
     """
     # generate prediction outputs, logits has shape BCHW[D]
     if logits.shape[1] == 1:
-        return (logits[:, 0] >= 0).int()  # for binary segmentation threshold on channel 0
+        return (logits >= 0).int()  # for binary segmentation threshold on channel 0
     else:
         return logits.max(1)[1]  # take the index of the max value along dimension 1
```

Comment:
line 57 will still remove the channel

Reply:
1. I think this logic is for "Argmax" but used wrong API (max) here, right?
    I changed to argmax in the latest patch.
2. I think it's still not correct even with argmax, it can support multi-labels segmentation?
Thanks.

Reply:
Will remove prediction from UNet output in other PR later.